src/js/tabs/receiver.js

```diff
@@ -381,42 +390,23 @@ TABS.receiver.initialize = function (callback) {
             $('select[name="rcSmoothing-input-derivative-select"]').change(function () {
                 if ($(this).val() == 0) {
                     $('.tab-receiver .rcSmoothing-derivative-cutoff').hide();
-                    RX_CONFIG.rcSmoothingDerivativeCutoff = 0;
+                    rcSmoothingDerivativeNumberElement.val(0);
                 }
                 if ($(this).val() == 1) {
                     $('.tab-receiver .rcSmoothing-derivative-cutoff').show();
-                    rcSmoothingnDerivativeNumberElement.val(RX_CONFIG.rcSmoothingDerivativeCutoff);
+                    rcSmoothingDerivativeNumberElement.val(RX_CONFIG.rcSmoothingDerivativeCutoff);
                 }
-            });
-
-            rcSmoothingnNumberElement.change(function () {
-                RX_CONFIG.rcSmoothingInputCutoff = $(this).val();
-            });
-            rcSmoothingnNumberElement.val(RX_CONFIG.rcSmoothingInputCutoff);
```

Comment:
I am slightly worried about removing this line - have you verified that the value for `rcSmoothingnNumberElement` does not have to be set here?

Reply:
@mikeller I set it on line 365 next to `rcSmoothingDerivativeNumberElement` before all change functions

Reply:
Ah, now I see. That's cleaner, good call!